Script Analysis Examples/Dataspace Analysis/DataSpace_Default_Analysis.ipynb

```diff
@@ -30,7 +30,6 @@
     "import scrapbook as sb\n",
```

Comment:
@vignesh-ni Make sure that this notebook is working as expected for actual data.

Reply:
Checked